client/src/app/common/Header/HeaderMain.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
I think, that I have already fixed this. So remove these lines

Reply:
+

Reply:
done